core/transfer/src/main/java/org/eclipse/dataspaceconnector/transfer/core/transfer/AddProvisionedResourceCommandHandler.java

```diff
@@ -0,0 +1,44 @@
+/*
+ *  Copyright (c) 2022 Microsoft Corporation
+ *
+ *  This program and the accompanying materials are made available under the
+ *  terms of the Apache License, Version 2.0 which is available at
+ *  https://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  SPDX-License-Identifier: Apache-2.0
+ *
+ *  Contributors:
+ *       Microsoft Corporation - initial API and implementation
+ *
+ */
+package org.eclipse.dataspaceconnector.transfer.core.transfer;
```

Comment:
```suggestion
package org.eclipse.dataspaceconnector.transfer.core.command.handlers;
```
to keep consistency with the other handlers